src/io/windows.zig

```diff
@@ -956,7 +956,7 @@ pub const IO = struct {
 
     pub const INVALID_FILE = os.windows.INVALID_HANDLE_VALUE;
 
-    fn open_file_handle(relative_path: []const u8, method: enum { create, open }) !os.fd_t {
+    fn open_file_handle(relative_path: []const u8, method: enum { create, open, open_wait }) !os.fd_t {
```

Comment:
Just curious, but why not `.deinit()` the `IO` to unlock the data file before starting the new process?

Reply:
No reason - that's a great idea! I think I should be able to deinit the replica and storage; afaik it's the port and the data file that we need to ensure are free.

(I was using the file as a proxy for both, here)